tests/components/__snapshots__/top_bar.test.ts.snap

```diff
@@ -513,7 +513,7 @@ exports[`TopBar component can set cell format 1`] = `
               class="o-composer w-100 text-start"
               contenteditable="true"
               spellcheck="false"
-              style="padding:5px 0px 5px 8px; max-height:100px; line-height:24px; height:fit-content;"
+              style="padding:5px 0px 5px 8px; max-height:100px; line-height:24px; height:34px;"
```

Comment:
Since we create the parent component of topbar without typing  the props, we were actually instanciating a top bar composer with its `focus` props as a boolean (instead of  `inactive`, `cellFocus`, `contentFocus`)  and a lot of code, specifically https://github.com/odoo/o-spreadsheet/blob/master/src/components/composer/top_bar_composer/top_bar_composer.ts#L39 ) tests against `inactive` which of course does not match a boolean. I checked that in thie specific snapshot, the composer is closed,  so the style  was wrong in the previous snapshot